server/src/models/species.es6

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const Sequelize = require('sequelize');
+const url = require('url');
+
+const sequelizeOptions = {
+  dialect: url.parse(process.env.DATABASE_URL, true).protocol.split(':')[0]
+};
+
+if (
+  url.parse(process.env.DATABASE_URL, true).hostname !== 'localhost' &&
+  url.parse(process.env.DATABASE_URL, true).hostname !== 'fs-intake-postgres'
+) {
+  sequelizeOptions.dialectOptions = {
+    ssl: true
+  };
+}
+
+const sequelize = new Sequelize(process.env.DATABASE_URL, sequelizeOptions);
+
+module.exports = sequelize.define(
+  'species',
+  {
+    id: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+      field: 'id',
+      allowNull: false
+    },
+    name: {
+      type: Sequelize.STRING,
+      field: 'name',
+      allowNull: false
+    },
+    webUrl: {
```

Comment:
is this for the photo? If so let's rename this field `photoUrl`

Reply:
@lauraGgit They are plant profiles (typically PDF links) from USDA. They aren't used on the front-end currently.

Reply:
@lpendem for accessibility reasons, PDF links don't seem like a great idea. If they are not being used in the front-end we should remove this.

Reply:
removed in sprint-2